h2o-docs/src/product/flow/packs/examples/XGBoost_Example.flow

```diff
@@ -0,0 +1,93 @@
+    },
+    {
+      "type": "md",
+      "input": "### Inspecting the Data Set\n\n Clicking on the **View** button opens the frame summary where you can examine the column summaries. "
+    },
+    {
+      "type": "cs",
+      "input": "getFrameSummary \"dataset_diabetes.hex\""
+    },
+    {
+      "type": "md",
+      "input": "### Inspecting the Selected Column\n\n Clicking on one of the colums you can examine the distribution of values of the specified column.\n Let's try that for the **readmitted** column."
+    },
+    {
+      "type": "cs",
+      "input": "getColumnSummary \"dataset_diabetes.hex\", \"readmitted\""
+    },
+    {
+      "type": "md",
+      "input": "### Splitting the Frame\n\n Clicking on **Split** action in the frame summary you can split the frame to several frames specifying the distribution of portions of the original dataset.\nLet's split the frame in 25:75 ratio and use the newly created frames as training and validation frame."
+    },
+    {
+      "type": "cs",
+      "input": "splitFrame"
+    },
+    {
+      "type": "cs",
+      "input": "splitFrame \"dataset_diabetes.hex\", [0.75], [\"frame_0.750\",\"frame_0.250\"], 198247"
+    },
+    {
+      "type": "md",
+      "input": "### Building a Model\n\n0. Once data are parsed, click the **View** button, then click the **Build Model** button. \n0. Select `XGBoost` from the drop-down **Select an algorithm** menu, then click the **Build model** button. \n0. Select the splited data set `frame_0.750` as the **training_frame** and the `frame_0.250` as the **validation_frame**. 0. From the **Ignored_columns** section, select the columns to ignore. For this example, ignore the `encounter_id` and `patient_nbr` columns as we don't want to build model based on these columns. \n0. From the drop-down **response_column** list, select column 1 (`readmitted`).  \n0. Make sure you set the correct value for **backend** and possibly **gpu_id** to run the computation on GPU if available. \n0. Click the **Build Model** button."
```

Comment:
Edit: "as we don’t want to build model based on these columns. " change to "as we don’t want to build model using ID columns as predictors."